docs/developer/transform.md

```diff
@@ -15,6 +15,16 @@ This page shows how to:
 
 After that, you might want to make your code transform into an [Autofix](autofix.md).
 
+There's also an important capability called "dry run transform":
+Performing a transform on a branch, and then either raising either a PR or
+an issue, depending on build success or failure. This allows us to
+safely apply transforms across many repositories. There's a simple wrapper
+function to enable this.
```

Comment:
This is a sentence needing more information, but [the original text](https://github.com/atomist/docs/blob/bdada7b458e833b44a5458ec1eafdf4e663329dd/docs/developer/sdm-concepts.md#dry-run-transforms) also abruptly ended in this way. What is the simple wrapper function we can link to here?

Reply:
It originally referred to this one I think: https://github.com/atomist/sdm/blob/0a8d8f488b83bfb6793f64c08870c33ea4c51699/lib/api-helper/command/transform/chattyDryRunAwareEditor.ts#L54

but @cdupuis may have info on a better way to do it now